src/Sulu/Bundle/AdminBundle/Resources/js/components/Number/tests/Number.test.js

```diff
@@ -1,40 +1,44 @@
 // @flow
 import React from 'react';
-import {render, shallow} from 'enzyme';
+import {render, fireEvent, screen} from '@testing-library/react';
```

Comment:
would be nice to use the `@testing-library/user-event` package instead of `fireEvent` 🙂 see https://github.com/sulu/sulu/pull/6783 for some examples